cli/cmd/plugin/ui/web/tanzu-ui/src/shared/services/awsDeployment.tsx

```diff
@@ -34,7 +34,7 @@ const useAwsDeployment = () => {
             createCloudFormationStack: false,
             clusterName: awsState.data.CLUSTER_NAME,
             controlPlaneFlavor: awsState.data.CLUSTER_PLAN,
-            controlPlaneNodeType: awsState.data.CONTROL,
+            controlPlaneNodeType: 'c5d.xlarge',
```

Comment:
I'm guessing this is a temporary hard-coding? May I suggest a `TODO`?

Reply:
Yes, I am going to try to resolve this today so instance type does get set from cluster settings selection.